pybotters/models/gmocoin.py

```diff
@@ -0,0 +1,616 @@
+            timestamp=parse_datetime(data.get("timestamp"))
+            if data.get("timestamp")
+            else datetime.now(timezone.utc),
+        )
+
+
+class GmoCoinDataStoreInterface(DataStoreInterface):
+    def _init(self) -> None:
+        self.create("orderbook", datastore_class=OrderBookStore)
+        self.create("orders", datastore_class=OrderStore)
+        self.create("positions", datastore_class=PositionStore)
+        self.create("executions", datastore_class=ExecutionStore)
+        self.create("position_summary", datastore_class=PositionSummaryStore)
+
+    async def initialize(self, *aws: Awaitable[aiohttp.ClientResponse]) -> None:
+        for f in asyncio.as_completed(aws):
+            resp = await f
+            data = await resp.json()
+            if (
+                resp.url.path == "/private/v1/latestExecutions"
+                and "list" in data["data"]
+            ):
+                self.executions._onresponse(
+                    MessageHelper.to_executions(data["data"]["list"])
+                )
+            if resp.url.path == "/private/v1/activeOrders" and "list" in data["data"]:
+                self.orders._onresponse(MessageHelper.to_orders(data["data"]["list"]))
+            if resp.url.path == "/private/v1/openPositions" and "list" in data["data"]:
+                self.positions._onresponse(
+                    MessageHelper.to_positions(data["data"]["list"])
+                )
+            if (
+                resp.url.path == "/private/v1/positionSummary"
+                and "list" in data["data"]
+            ):
+                self.position_summary._onresponse(
+                    MessageHelper.to_position_summaries(data["data"]["list"])
+                )
+
+    def _onmessage(self, msg: Item) -> None:
+        if "channel" in msg:
+            msg_type = MessageType[msg.get("msgType", MessageType.NONE.name)]
+            channel: Channel = Channel.from_str(msg["channel"])
+            # Public
+            if channel == Channel.ORDER_BOOKS:
+                self.orderbook._onmessage(MessageHelper.to_orderbook(msg))
+            # Private
+            elif channel == Channel.EXECUTION_EVENTS:
+                asyncio.create_task(
+                    self.orders._onexecution(MessageHelper.to_execution(msg))
+                )
+                asyncio.create_task(
+                    self.executions._onmessage(MessageHelper.to_execution(msg))
+                )
+            elif channel == Channel.ORDER_EVENTS:
+                asyncio.create_task(self.orders._onmessage(MessageHelper.to_order(msg)))
+            elif channel == Channel.POSITION_EVENTS:
+                self.positions._onmessage(MessageHelper.to_position(msg), msg_type)
+            elif channel == Channel.POSITION_SUMMARY_EVENTS:
+                self.position_summary._onmessage(MessageHelper.to_position_summary(msg))
+
+    @property
+    def orderbook(self) -> OrderBookStore:
```

Comment:
```suggestion
    def orderbooks(self) -> OrderBookStore:
```
GMOもAPIドキュメント上エンドポイント名やチャンネル名が `orderbooks` になっているので合わせて頂くのが好ましいです。